src/main/kotlin/com/teamdev/jxbrowser/gradle/JxBrowserExtension.kt

```diff
@@ -150,14 +158,18 @@ public open class JxBrowserExtension(private val project: Project) {
 
     private fun currentPlatform(): Provider<String> {
         val platformMap =
-            mapOf(
-                Pair({ isX64Bit() && isWindows() }, win64),
-                Pair({ isX64Bit() && isLinux() }, linux64),
-                Pair({ isX64Bit() && isMac() }, mac),
-                Pair({ is32Bit() && isWindows() }, win32),
-                Pair({ isArm() && isLinux() }, linuxArm),
-                Pair({ isArm() && isMac() }, macArm),
+            mutableMapOf(
+                { isX64Bit() && isWindows() } to win64,
+                { isX64Bit() && isLinux() } to linux64,
+                { isX64Bit() && isMac() } to mac,
+                { is32Bit() && isWindows() } to win32,
+                { isArm() && isLinux() } to linuxArm,
+                { isArm() && isMac() } to macArm,
             )
+        if (version.startsWith("8")) {
+            // As for now, Windows ARM is only supported in JxBrowser 8.
+            platformMap.put({ isArm() && isWindows() }, win64Arm)
```

Comment:
I think we should always add this entry to `platformMap`. If the current version is not 8, then we will throw an exception later inside the `artifact()` method with a clear message to the user.